launch.go

```diff
@@ -147,15 +147,24 @@ func createWorkspace(rootDir string, srcPaths ...string) (Workspace, error) {
 	return w, nil
 }
 
-func createMetaSpace(metaSpace string) error {
+func createMetaSpace(metaSpace string, isLocal bool) error {
```

Comment:
I think if we can pass `meta` from `sd-local`, we should create a `metafile` to use the passed `meta` in a local build.
it's referenced this line: https://github.com/screwdriver-cd/screwdriver/pull/1884/files#diff-49f99a06253f0a9a292e89f6a469b9e4R46

Reply:
I forgot that `sd-local` support `SD_META_DIR` and `SD_META_PATH`.
I fixed it.